tests/unit/serve/executors/test_executor.py

```diff
@@ -75,6 +77,34 @@ def test_executor_import_with_external_dependencies(capsys):
     assert 'hello' in out
 
 
+def test_executor_with_pymodule_path():
+    with pytest.raises(FileNotFoundError):
+        ex = Executor.load_config(
+            '''
+        jtype: BaseExecutor
+        metas:
+            py_modules:
+                - jina.no_valide.executor
+        '''
+        )
+
+    import sys
+
+    sys.path.append(cur_dir)
```

Comment:
do this with an independent file and in a fixture separated from the test logic

Reply:
no need anymore, we can directly use the tests folder as the module root.